backend/services/esweService.ts

```diff
@@ -162,16 +164,74 @@ export default class EsweService {
     return createFlaggedContent(content)
   }
 
-  async getLearnerEmployabilitySkills(nomisId: string): Promise<LearnerEmployabilitySkills> {
-    let result: LearnerEmployabilitySkills = null
+  async getLearnerEmployabilitySkills(nomisId: string): Promise<EmployabilitySkills> {
```

Comment:
This could do with more tests - the existing tests don't seem to demonstrate how the skills are reduced.